sdcm/cluster.py

```diff
@@ -3682,8 +3682,8 @@ def get_cluster_manager(self):
         mgr_cluster = manager_tool.get_cluster(cluster_name)
         if not mgr_cluster:
             self.log.debug("Could not find cluster : {} on Manager. Adding it to Manager".format(cluster_name))
-            targets = [[n, n.ip_address] for n in self.nodes]
-            mgr_cluster = manager_tool.add_cluster(name=cluster_name, host=targets[0], disable_automatic_repair=True,
+            target = self.nodes[0].ip_address
```

Comment:
can you move it it into it's own commit or PR ?

Reply:
Yep. I just try to pass thru provision test here.

Reply:
Done: https://github.com/scylladb/scylla-cluster-tests/pull/2356